package.json

```diff
@@ -132,7 +132,12 @@
     "rollup-plugin-node-resolve": "^4.2.3",
     "staged-git-files": "^1.3.0",
     "stylus": "^0.54.8",
-    "testcafe": "^2.6.2",
-    "testcafe-browser-provider-saucelabs": "^1.9.0"
+    "testcafe": "^3.5.0",
+    "testcafe-browser-provider-saucelabs": "^2.0.0"
```

Comment:
We won't be able to upgrade to v3.0.0 of this package until this bug will be fixed: https://github.com/DevExpress/testcafe-browser-provider-saucelabs/issues/95. Our tests trigger a timeout on GH when using v3.0.0.

Reply:
good to know!